src/thor/triplegbuilder.cc

```diff
@@ -1632,7 +1611,7 @@ TripLegBuilder::Build(const AttributesController& controller,
 
     // Set shape attributes
     SetShapeAttributes(controller, graphtile, directededge, costing, trip_shape.begin() + begin_index,
-                       trip_shape.end(), trip_path, second_of_week, edge_pct);
+                       trip_shape.end(), trip_path, second_of_week, length_pct);
```

Comment:
again we forgot to do partial shape attributes. we had already computed the percenage for partial shape so i reused it